### PR TITLE
Add future work write-up

### PR DESCRIPTION
**Pull request by stulle123**
Hi,

This PR adds a write-up about future work.

Cheers,

stulle123

### CODE REVIEW

File: future_work/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - Enable technical support access in `service_manager.pyc` by modifying the `techsupport_access` method and [rebuild/reflash the rootfs](../README.md#flashing-the-nand-images)
+- Try a shell escape with [ssh](https://gtfobins.github.io/gtfobins/ssh/), [telnet](https://gtfobins.github.io/gtfobins/telnet/), [tcpdump](https://gtfobins.github.io/gtfobins/tcpdump/), [find](https://gtfobins.github.io/gtfobins/find/), [diff](https://gtfobins.github.io/gtfobins/diff/) or [grep](https://gtfobins.github.io/gtfobins/grep/)
+- Read password values with `get` command from config file(s)
+- Edit files with the `edit` command --> Add backdoor to `/etc/rc` script
+- In the router's rootfs grep for possible command injection flaws:
+```
+$ grep -iE "os.system|os.popen|commands.getstatusoutput|commands.getoutput|commands.getstatus|subprocess.call|subprocess.run|subprocess.Popen|pty.spawn|execfile|exec|eval"
+```
+
+## Miscellaneous
```
Comment (vegantransistor) [inline future_work/README.md:19]: what about the USB port?

Comment (stulle123) [reply future_work/README.md:19]: You mean firmware upgrade via USB? USB stack fuzzing?

Comment (vegantransistor) [reply future_work/README.md:19]: yes, everything related to USB attack surface.

Comment (stulle123) [reply future_work/README.md:19]: Added.

**Pull request merged**